DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -32,6 +32,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
+        if (class_exists('Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface')) {
+            $loader->load('psr7.xml');
+        }
```

Comment:
Currently if you add the bridge and doesn't typehint a controller nothing will happen, and if you add the bridge, add the typehint and don't want the param conversion it's kinda strange.